scripts/install/install-dependencies.sh

```diff
@@ -563,6 +481,7 @@ function install_conda() {
   # Where CONDA_INSTALL_DIR is where Anaconda or miniconda was installed.
   # We do the first step here. Later we'll activate the PANOPTES environment.
 
+  # shellcheck disable=SC1090
```

Comment:
What check is this disabling? Just curious since there are a few of them.

Reply:
Shellcheck supports checks that span multiple files, but only if it knows where that other file is, and it is complaining that it can't figure out the value of CONDA_SH and so doesn't know where conda is located.

Since we don't write the conda code, I'm disabling that feature here.